src/com/organizationmanager/OrganizationManager.java

```
@@ -0,0 +1,394 @@
+package com.organizationmanager;
+
+import javax.xml.parsers.*;
+import javax.xml.transform.*;
+import javax.xml.transform.dom.*;
+import javax.xml.transform.stream.*;
+import org.w3c.dom.*;
+import java.io.*;
+import java.nio.file.*;
+import java.time.*;
+import java.time.format.DateTimeFormatter;
+import java.util.*;
+import java.util.stream.*;
+
+public class OrganizationManager {
+    private final Set<Organization> organizations;
+    private final Queue<String> commandHistory;
+    private final Set<Path> activeScripts;
+    private final Path defaultSavePath;
+    private long nextId;
+
+    public OrganizationManager(Set<Organization> organizations, String saveFileName) {
+        this.organizations = new TreeSet<>(organizations);
+        this.commandHistory = new ArrayDeque<>(8);
+        this.activeScripts = new HashSet<>();
+        this.defaultSavePath = Paths.get(saveFileName).toAbsolutePath();
+        this.nextId = organizations.stream()
+                .mapToLong(Organization::getId)
+                .max().orElse(0) + 1;
+    }
+
+    public void startInteractiveMode() {
+        try (Scanner scanner = new Scanner(System.in)) {
+            System.out.println("Organization Manager started. Type 'help' for commands.");
+            while (true) {
+                try {
+                    System.out.print("> ");
+                    String input = scanner.nextLine().trim();
+                    if (input.isEmpty()) continue;
+
+                    processCommand(input, scanner);
+                } catch (NoSuchElementException e) {
+                    System.out.println("\nExiting...");
+                    return;
+                } catch (Exception e) {

```

Don't do catch all exceptions